lib/ace/autocomplete/papeeria/tex_completer.coffee

```diff
@@ -0,0 +1,15 @@
+define (require, exports, module) ->
+  exports.getCompletions = (editor, session, pos, prefix, callback) ->
+    wordList = [ '\\item' ]
+    callback null, wordList.map((word) ->
```

Comment:
let's agree that we do use braces in function calls in CoffeeScript. Otherwise code becomes difficult to understand